test/e2e/kubevirt.go

```diff
@@ -0,0 +1,628 @@
+				CompletionTimeoutPerGiB: pointer.Int64(1),
+				BandwidthPerMigration:   &bandwidthPerMigration,
+				Selectors: &kvmigrationsv1alpha1.Selectors{
+					VirtualMachineInstanceSelector: kvmigrationsv1alpha1.LabelSelector{
+						"test-live-migration": "post-copy",
+					},
+				},
+			},
+		}
+		if td.mode == kubevirtv1.MigrationPostCopy {
+			_, err = kvcli.MigrationPolicy().Create(context.TODO(), forcePostCopyMigrationPolicy, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			defer func() {
+				Expect(kvcli.MigrationPolicy().Delete(context.TODO(), forcePostCopyMigrationPolicy.Name, metav1.DeleteOptions{})).To(Succeed())
+			}()
+		}
+
+		By("Creating a test pod on both selected worker nodes")
+		podWorker1 = composeAgnhostPod(
+			"testpod-ovn-worker",
+			namespace,
+			"ovn-worker",
+			"/bin/bash", "-c", "/agnhost netexec --http-port 8000")
+		podWorker2 = composeAgnhostPod(
+			"testpod-ovn-worker2",
+			namespace,
+			"ovn-worker2",
+			"/bin/bash", "-c", "/agnhost netexec --http-port 8000")
+		_ = fr.PodClient().CreateSync(podWorker1)
+		_ = fr.PodClient().CreateSync(podWorker2)
+
+		By("Waiting until both pods have an IP address")
+		Eventually(func() error {
+			var err error
+			podWorker1, err = fr.ClientSet.CoreV1().Pods(fr.Namespace.Name).Get(context.TODO(), podWorker1.Name, metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			if podWorker1.Status.PodIP == "" {
+				return fmt.Errorf("pod %s has no valid IP address yet", podWorker1.Name)
+			}
+			podWorker2, err = fr.ClientSet.CoreV1().Pods(fr.Namespace.Name).Get(context.TODO(), podWorker2.Name, metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			if podWorker2.Status.PodIP == "" {
+				return fmt.Errorf("pod %s has no valid IP address yet", podWorker2.Name)
+			}
+			return nil
+		}).
+			WithTimeout(time.Minute).
+			WithPolling(time.Second).
+			Should(Succeed())
+
+		By("Create virtual machines")
+		vmLabels := map[string]string{}
+		if td.mode == kubevirtv1.MigrationPostCopy {
+			vmLabels = forcePostCopyMigrationPolicy.Spec.Selectors.VirtualMachineInstanceSelector
+		}
+		vms, err := composeVMs(td.numberOfVMs, vmLabels)
+		Expect(err).ToNot(HaveOccurred())
+		for _, vm := range vms {
+			vm, err = kvcli.VirtualMachine(namespace).Create(vm)
+			Expect(err).ToNot(HaveOccurred())
+		}
+		for _, vm := range vms {
+			Eventually(func() bool {
+				vm, err = kvcli.VirtualMachine(namespace).Get(vm.Name, &metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				return vm.Status.Ready
+			}).WithPolling(time.Second).WithTimeout(5 * time.Minute).Should(BeTrue())
+		}
+
+		wg.Add(int(td.numberOfVMs))
+		for _, vm := range vms {
+			go runTest(td, vm)
+		}
+		wg.Wait()
+	},
+		Entry("with pre-copy should keep connectivity", liveMigrationTestData{
+			mode:                 kubevirtv1.MigrationPreCopy,
+			numberOfVMs:          1,
+			numberOfMigrations:   2,
+			checkNetworkPolicies: true,
```

Comment:
what's the point of this attribute if you're always setting it to true ? 

Are you planning to add more tests ?

Reply:
It was giving me problems at github actions, but is not the case anymore so we don't need the knob, I will remove it.